main/Card/index.tsx

```diff
@@ -83,6 +86,8 @@ interface Props {
   dividerStyle?: ViewStyle;
   outlined?: boolean;
   raised?:boolean;
+  titleNumberOfLine?: number;
+  subTitleNumberOfLine?: number;
```

Comment:
```suggestion
  titleNumberOfLines?: number;
  subTitleNumberOfLines?: number;
```
to give unity with TextProps.numberOfLines